hail/python/hail/genetics/pedigree.py

```diff
@@ -161,27 +169,18 @@ class Pedigree(object):
     """
 
     def __init__(self, trios):
-        self._jrep = Env.hail().methods.Pedigree(jindexed_seq([t._jrep for t in trios]))
-        self._trios = trios
-
-    @classmethod
-    def _from_java(cls, jrep):
-        ped = Pedigree.__new__(cls)
-        ped._jrep = jrep
-        ped._trios = None
-        super(Pedigree, ped).__init__()
-        return ped
+        self._trios = tuple(trios)
 
     def __eq__(self, other):
-        return isinstance(other, Pedigree) and self._jrep == other._jrep
+        return isinstance(other, Pedigree) and self._trios == other._trios
 
     def __hash__(self):
-        return self._jrep.hashCode()
+        return hash(self._trios)
```

Comment:
It might be good to add `__iter__` methods so a user can iterate over each trio in the pedigree.